com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/SharedCode.template.hlsl

```diff
@@ -17,6 +17,11 @@ FragInputs BuildFragInputs(VaryingsMeshToPS input)
     $FragInputs.texCoord3:          output.texCoord3 = input.texCoord3;
     $FragInputs.color:              output.color = input.color;
 
+    #ifdef HAVE_VFX_MODIFICATION
+    // FragInputs from VFX come from two places: Interpolator or CBuffer.
+    $splice(VFXSetFragInputs)
```

Comment:
Is it possible to re-use the new custom interpolators feature for this? VFX interpolants are basically just custom interpolators...

Reply:
you mean merge it in CustomInterpolatorVaryingsToFragInputs ? If we can, we should